public/index.ts

```diff
@@ -1339,11 +1339,12 @@ function main() {
     view3D.setMaxProjectMode(myState.volume, mp);
   };
   renderModeSelect?.addEventListener("change", ({ currentTarget }) => {
-    if ((currentTarget as HTMLOptionElement)!.value === "PT") {
+    const target = (currentTarget as HTMLOptionElement)!;
```

Comment:
Curious how smart typescript and/or babel and/or V8 are about deleting a simple alias like this